charts/greptimedb-cluster/templates/grafana-dashboards-configmap.yaml

```diff
@@ -6,6 +6,8 @@ kind: ConfigMap
 metadata:
   name: {{ $value }}
   namespace: {{ $root.Release.Namespace }}
+  labels:
+    {{- include "greptimedb-cluster.labels" . | nindent 4 }}
```

Comment:
Hey @nlamirault  I believe this should be:
```
  labels:
    {{- include "greptimedb-cluster.labels" $root | nindent 4 }}
```
`{{- $root := . -}}` has been defined in the line 2.